### PR TITLE
Add more subjects!

### PR DESCRIPTION
**Pull request by Matanlb**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Matanlb/TestReviewTool/pull/2" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -2,3 +2,6 @@
 
 ### Some Subject
 Text text text
+
+$$$ Subject
```
Comment (Matanlb) [inline README.md]: I think you tried to make a headline, use # instead  of $.

!resolve

File: README.md [hunk 2]
```diff
@@ -2,3 +2,6 @@
...
+Text and more text
```
Comment (Matanlb) [inline README.md]: Always and a line with a full stop.

!fix

File: README.md (hunk 1 above)
Comment (Matanlb) [inline README.md]: You are right...

!resolved

File: README.md (hunk 2 above)
Comment (Matanlb) [inline README.md]: Fine....

!fixed

**Pull request merged**